### PR TITLE
add marabou-cifar10 benchmarks

### PR DESCRIPTION
**Pull request by wu-haoze**

### CODE REVIEW

File: benchmarks/marabou-cifar10/generate_linf_robustness_query.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        val = image[i]
+        lb = max(0, val - epsilon)
+        ub = min(1, val + epsilon)
+        print("(assert (<= X_{} {})".format(i, ub), file=f)
+        print("(assert (>= X_{} {})".format(i, lb), file=f)
+
+    print( "; Output constraints",file=f)
+    for i in range(10):
+        if i != targetLabel:
+            print("(assert (>= Y_{} Y_{}))".format(targetLabel, i), file=f)
```
Comment (mnmueller) [inline benchmarks/marabou-cifar10/generate_linf_robustness_query.py:63]: Can you double-check that this is the specification you want to have verified. As the spec is describing a counterexample, every image that is correctly classified will lead to a SAT output immeadiately.

**Comment by stanleybak** [comment]
Yes, I agree with @mnmueller . I believe the spec you want to generate is something like:

```
(assert (or
    (and (>= Y_0 Y_5))
    (and (>= Y_1 Y_5))
    (and (>= Y_2 Y_5))
    (and (>= Y_3 Y_5))
    (and (>= Y_4 Y_5))
    (and (>= Y_6 Y_5))
    (and (>= Y_7 Y_5))
    (and (>= Y_8 Y_5))
    (and (>= Y_9 Y_5))
))
```

Please fix.

**Comment by wu-haoze** [comment]
Hi @mnmueller and @stanleybak, thanks for taking a look. 
Here I'm trying to check whether the classifier will mis-classify a set of images as a particular incorrect label. Therefore, the spec specifies that the (incorrect) **target label** is the max over all outputs. Sorry that this is not clear.
I updated the readme to clarify the property! Please let me know if it looks ok.

**Comment by stanleybak** [comment]
Ok sure. 

Your assert lines have mismatched parentheses:
`(assert (>= X_187 0.4115294163227081)`

please fix.

**Comment by wu-haoze** [comment]
Whoops, just fixed the script and the example specs with seed 0. Please check!

**Pull request merged**